### PR TITLE
Modification for 'Configure the user claims' in 'Configuring Email OTP'

### PR DESCRIPTION
**Pull request by AnuradhaSK**
## Purpose
Resolves https://github.com/wso2/docs-is/issues/1282

### CODE REVIEW

File: en/docs/learn/configuring-email-otp.md [hunk 1]
```diff
@@ -583,40 +583,24 @@ Follow the steps given below to update the user's email address.
...
-            `                             http://wso2.org/claims/identity/emailotp_disabled                           `
-        2.  **Display Name** :
-            `              DisableEmailOTP             `
-        3.  **Description:**
-            `              DisableEmailOTP             `
-        4.  **Mapped Attribute (s):** `              title             `
-        5.  **Supported by Default:** checked
-
-        ![filling-claim-fields](../assets/img/tutorials/filling-claim-fields.png)
+    3.  Click **Suport by Default** and **Update**.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-email-otp.md]: Select **Supported by Default** and click **Update**

File: en/docs/learn/configuring-email-otp.md [hunk 2]
```diff
@@ -583,40 +583,24 @@ Follow the steps given below to update the user's email address.
...
 
     4.  Click **Add**.
 
         To disable this claim for the admin user, navigate to **Users
         and Roles \> List** and click **Users.** Click on the **User
-        Profile** link corresponding to admin account and then click
-        **Disable EmailOTP.** This will disable the second factor
+        Profile** link corresponding to admin account and then write 
+        **false** in **Disable EmailOTP.** This will disable the second factor
         authentication for the admin user.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-email-otp.md]: ...and enter "false" as the value for the **Disable EmailOTP** field.

File: en/docs/learn/configuring-email-otp.md (hunk 1 above)
Comment (AnuradhaSK) [reply en/docs/learn/configuring-email-otp.md]: Addressed in dde107f

File: en/docs/learn/configuring-email-otp.md (hunk 2 above)
Comment (AnuradhaSK) [reply en/docs/learn/configuring-email-otp.md]: addressed in dde107f and 2532c49

**Pull request merged**